frontend/package.json

```diff
@@ -20,6 +20,7 @@
     "filepond-polyfill": "^1.0.2",
     "hoist-non-react-statics": "^3.2.1",
     "jest-localstorage-mock": "^2.2.0",
+    "jsonwebtoken": "^8.5.1",
```

Comment:
Do you still need this package?

Reply:
Oh, not anymore. That logic/dependency was moved to the python-backend